src/vs/workbench/browser/parts/editor/media/multieditortabscontrol.css

```diff
@@ -50,7 +50,7 @@
 	display: none;
 }
 
-.monaco-workbench .part.editor > .content .editor-group-container > .title .tab:not(.active) .tabs-and-actions-container.tabs-border-bottom::after {
```

Comment:
It makes no sense to have a `.tab` before `.tabs-and-actions-container`. This would never apply

Reply:
This used to be `.monaco-workbench .part.editor > .content .editor-group-container > .title > .tabs-and-actions-container.tabs-border-bottom::after {
` in earlier versions, is the removal of `>` intentional?